pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -198,13 +198,15 @@ public Object handle(Request request, Response response) {
             List<Tissue> tissueIds =
                     oncHistoryDetailDaoImpl.getRandomOncHistoryDetail(randomOncHistoryDetailId, ddpInstance.getName()).getTissues();
             Integer tissueId = null;
-            if (tissueIds != null && !tissueIds.isEmpty()) {
+            //if the random onc history detail has a tissue with a collaborator sample id, use that tissue
+            if (!tissueIds.isEmpty()) {
                 Optional<Tissue> tissue = tissueIds.stream().filter(tissue1 ->
                         tissue1 != null && StringUtils.isNotBlank(tissue1.getCollaboratorSampleId())
                 ).findAny();
                 tissueId = tissue.isPresent() ? tissue.get().getTissueId() : null;
             }
-            if (tissueId != null || tissueIds.isEmpty()) {
```

Comment:
The conditions were wrong in which it checked for existing eligible tissues, but then it only created a tissue if an existing tissue was found, which is the wrong behavior